### PR TITLE
Load fat jar but do not analyse dependencies [TG-9521] 

### PR DESCRIPTION
**Pull request by owen-mc-diffblue**

### CODE REVIEW

File: diffblue-base.json [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {
   "phaseBase": {
-    "javaVersion": 6
+    "javaVersion": 6,
+    "preferDepsJar": true,
+    "context-include": "com.jsoniter"
   }
 }
```
Comment (majakusber) [inline diffblue-base.json]: :pick: Missing extra line

**Review by majakusber — APPROVED** [review]
Thank you for removing the old files too :+1:

File: diffblue-base.json (hunk 1 above)
Comment (peterschrammel) [reply diffblue-base.json]: You need to context-include `java.` and `org.cprover.` too.

Comment (owen-mc-diffblue) [reply diffblue-base.json]: @svorenova I don't understand what you mean
@peterschrammel Thanks -  I didn't realise that. How do I separate values? Or do I give the flag multiple times?

Comment (peterschrammel) [reply diffblue-base.json]: `"context-include": [ "com.jsoniter.", "java.", "org.cprover." ]`

**Comment by owen-mc-diffblue** [comment]
Oops, I based this branch on an old version of `add-jacoco` by mistake. I've fixed it now.

**Review by JohnLBergqvist — APPROVED** [review]
Can you squash before merging please.

**Comment by owen-mc-diffblue** [comment]
@JohnLBergqvist Sure, if that's the way you want it on this repo. Do you mean squashing all the commits into one, or just the last two?

**Comment by JohnLBergqvist** [comment]
@owen-mc-diffblue Just the last two :)

**Comment by majakusber** [comment]
@JohnLBergqvist Why? I wouldn't do that, it's two independent changes, there's no reason to have them in one commit.

**Comment by JohnLBergqvist** [comment]
@svorenova I was referring to the commit that adds a newline to the `diffblue-base.json` file. That can be squashed into the previous commit (which updates the file).

**Comment by owen-mc-diffblue** [comment]
@JohnLBergqvist The 2nd commit doesn't create the file. It already exists, with `"javaVersion": 6` in it.

**Comment by JohnLBergqvist** [comment]
@owen-mc-diffblue What happens if the fat jar doesn't actually contain all of the classes in the thin jar? Because we've seen some pretty funky configs where that can happen, in the wild.

**Pull request merged**